spec/serializers/show_serializer_spec.rb

```diff
@@ -32,7 +32,8 @@
           'name' => an_instance_of(String)
         },
         'authors' => an_instance_of(Array),
-        'image' => an_instance_of(String)
+        'image' => an_instance_of(String),
+        'authors_as_sentence' => an_instance_of(String)
```

Comment:
and this here `'top_story' => an_instance_of(FalseClass),`

Reply:
Added!